styles/cspace/SearchResultTile.css

```diff
@@ -11,6 +11,7 @@
   background-color: white;
   color: inherit;
   font-size: 0.9rem;
+  border: 1px solid inputBorderColor;
```

Comment:
I think I'd rather have this be `transparent` instead of `inputBorderColor`, to be more subtle.